src/lib/services/feature-toggle-service.ts

```diff
@@ -1350,6 +1350,15 @@ class FeatureToggleService {
         newProject: string,
         createdBy: string,
     ): Promise<void> {
+        const changeRequestEnabled =
+            await this.changeRequestAccessReadModel.isChangeRequestsEnabledForProject(
+                newProject,
+            );
+        if (changeRequestEnabled) {
```

Comment:
what happens if the it is an API user with "skip change request" permissions?

Reply:
Currently it's still blocked. SKIP_CHANGE_REQUEST was mainly created for emergency situations/monitoring scripts to quickly turn off the kill switch without going through the change request process. The question is should we do the same for cloning a toggle/moving a toggle between projects.